docs/compendium/substitute.md

```diff
@@ -185,6 +186,45 @@ additional metadata of a substituted ERC
 }
 ```
 
+Example 02 - in case the overlayfile has the same filename as one of the existing basefiles :
+
+```json
+200 OK
+
+{
+  "id": "oMMFn",
+  ...
+  "metadata": {
+      ...
+      "substituted": true,
+      "substitution": {
+          "base": "G92NL",
+          "overlay": "9fCTR",
+          "substitutionFiles": [
+            {
+              "base": "climate-timeseries.csv",
+              "overlay": "input.csv",
+              "filename": "overlay_input.csv"
+            }
+          ]
+      },
+      ...
+      },
+  ...
+}
+```
+
+### Response additional metadata
+
+- `substituted` - will be set `true`
+- `substitution` - object, specifying information about the substitution
+  - `base` - id of the base ERC
+  - `overlay` - id of the overlay ERC
+  - `substitutionFiles` - array of file substitutions specified by `base` and `overlay`
+    - `base` - filename of the file from the base ERC
+    - `overlay` - filename of the file from the overlay ERC
+    - `filename` - as seen in the examples above, `filename` will be created if there is a conflict with any basefilename and an overlayfilename. In this case the overlayfilename will get an additional "**overlay_**" prepended (see Example 02). *(optional add)*
+
 ## List substituted Compendia
```

Comment:
Nouns in the API paths should be singular, so so this must be `GET /api/v1/substitution`.

Also, the sentence "... have been substituted" missing a period at the end.